sys-apps/xdg-desktop-portal-kde/xdg-desktop-portal-kde-5.13.3.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit cmake-utils
+
+SRC_URI="mirror://kde/stable/plasma/${PV}/${P}.tar.xz"
+DESCRIPTION="A backend implementation for xdg-desktop-portal using Qt/KF5"
+HOMEPAGE="https://www.kde.org/workspaces/plasmadesktop/"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="wayland"
+
+DEPEND="
+	>=kde-frameworks/extra-cmake-modules-5.42.0
+	>=kde-frameworks/ki18n-5.42.0
+	>=kde-frameworks/knotifications-5.42.0
+	>=kde-frameworks/kcoreaddons-5.42.0
+	>=kde-frameworks/kwidgetsaddons-5.42.0
+	wayland? (
+		>=kde-frameworks/kwayland-5.42.0
+		)
+	>=dev-qt/qtcore-5.9.0
+	>=dev-qt/qtgui-5.9.0
+	>=dev-qt/qtwidgets-5.9.0
+	>=dev-qt/qtdbus-5.9.0
+"
```

Comment:
I seem to miss some deps mentioned in cmake file (eg. concurrent), but I am not up-to-date on qt/kde.